echo/.devcontainer/docker-compose.yml

```
@@ -101,10 +118,16 @@ services:
       - STORAGE_S3_SECRET=dembrane
       - STORAGE_S3_BUCKET=dembrane
       - STORAGE_S3_ENDPOINT=http://minio:9000
+      - NEO4J_URL=bolt://neo4j:7687
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)